.vs/config/applicationhost.config

```diff
@@ -0,0 +1,1027 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
@uffebjorklund is this file required to be checked in or should everything in `.vs` be ignored?